nodestream/cli/operations/run_project_cookiecutter.py

```diff
@@ -15,13 +15,17 @@ def __init__(self, project_name, project_path, database):
     async def perform(self, _: NodestreamCommand):
         from ..application import get_version
 
+        version = get_version()
+        semver_non_patch_version = ".".join(version.split(".")[:2])
+
         cookiecutter(
             PROJECT_COOKIECUTTER_URL,
             no_input=True,
             output_dir=self.project_path,
             extra_context={
                 "project_name": self.project_name,
                 "database": self.database,
-                "nodestream_version": get_version(),
+                "nodestream_version": version,
+                "nodestream_semver_non_patch_version": semver_non_patch_version,
```

Comment:
What does this code portion do again? I understand this part is relevant to the Neptune plugin introduction.

Reply:
`nodestream_semver_non_patch_version` would equal `0.12` for the upcoming release. It will be used by the template to get the version `0.12` for the associated plugin. Without this, we'd only have `nodestream_version` which would equal something like `0.12.12` down the road. We don't want to release patch versions across all dbs every time we release the core library.

Reply:
Ahh okay makes sense, so it just takes 0.a out of 0.a.b. So only when a major 0.13 comes out well have to rev the plugins.